test/TwigEnvironmentFactoryTest.php

```diff
@@ -101,8 +100,8 @@ public function testCanSpecifyCacheDirectoryViaConfiguration()
     public function testAddsTwigExtensionIfRouterIsInContainer()
     {
         $twigExtensionFactory = new TwigExtensionFactory();
-        $serverUrlHelper = $this->prophesize(ServerUrlHelper::class)->reveal();
-        $urlHelper       = $this->prophesize(UrlHelper::class)->reveal();
+        $serverUrlHelper      = $this->prophesize(ServerUrlHelper::class)->reveal();
```

Comment:
Oh god, we went for alignment of assignments.
Context:
https://discourse.zendframework.com/t/poll-zend-coding-standard-2-aligning-statements-arrays-and-comment-tags/729